exporter/awsemfexporter/cwlog_client.go

```diff
@@ -175,9 +176,19 @@ func (client *cloudWatchLogClient) CreateStream(logGroup, streamName *string) (t
 	return "", nil
 }
 
-func newCollectorUserAgentHandler(buildInfo component.BuildInfo) request.NamedHandler {
+func newCollectorUserAgentHandler(buildInfo component.BuildInfo, logGroupName string) request.NamedHandler {
+	fn := request.MakeAddToUserAgentHandler(collectorDistribution, buildInfo.Version)
+	if matchContainerInsightsPattern(logGroupName) {
+		fn = request.MakeAddToUserAgentHandler(collectorDistribution, buildInfo.Version, "ContainerInsights")
+	}
 	return request.NamedHandler{
 		Name: "otel.collector.UserAgentHandler",
-		Fn:   request.MakeAddToUserAgentHandler(collectorDistribution, buildInfo.Version),
+		Fn:   fn,
 	}
 }
+
+func matchContainerInsightsPattern(logGroupName string) bool {
+	regexP := "^\\/aws\\/.*containerinsights\\/.*\\/(performance|prometheus)$"
```

Comment:
Just curious, why do you need to escape the forward slashes?  Wouldn't `"^/aws/.*containerinsights/.*/(performance|prometheus)$"` work?

Reply:
You're right, it was only required when I tested the regex in some web regex editor.
Made the update, thx.